pagina-practica/style.css

```diff
@@ -30,12 +30,164 @@
 }
 
 #img{
-    width: 700px;
-    height: 400px;
-    margin-left: 290px;
+
+    display: flex;
+    width: 600px;
+    height: 350px;
+    border: solid;
+    margin-top: 10px 0 10px ;
+ 
 
 }
 
 .home{
-    background: whitesmoke;
+    background: lavender;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.about{
+    
+    background-color: lavenderblush;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 100px;
+
+}
+
+.parrafo{
+    display: block;
+    align-items: column;
+    margin: 12px 300px 0 0;
+}
+#imagen-about{
```

Comment:
Why still using ID's as selectors?
https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4  read this please.